slf4j/src/main/scala-2/org/typelevel/log4cats/slf4j/LoggerNamePlatform.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.slf4j
+
+import org.typelevel.log4cats.slf4j.internal.GetLoggerMacros
+
+trait LoggerNamePlatform {
```

Comment:
Basically `Platform` suffix is used when dealing with `JVM`/ `JS`-specific things. Probably `Compat` is more suitable here.